### PR TITLE
Associated contributions to categories for each contributor

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Associated contributions to categories for each contributor
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | N/A

### CODE REVIEW

**Comment by Progi1984** [comment]
Waiting for @ttoine !

File: traces [hunk 1]
```diff
@@ -37,6 +37,204 @@ const GITHUB_CONTRIBUTORS_POINT = 'https://api.github.com/repos/{repository}/con
...
+        'PrestaShop/presthubot',
+        'PrestaShop/QANightlyResults',
+        'PrestaShop/core-weekly-generator',
+        'PrestaShop/email-templates-sdk',
+        'PrestaShop/fontmanager',
+        'PrestaShop/vagrant',
+    ],
+    'others' => [
+        'PrestaShop/prestashop.github.io',
+        'PrestaShop/solrsearch',
```
Comment (PierreRambaud) [inline traces:231]: is a module :thinking:

Comment (ttoine) [reply traces:231]: is it something we should archive? does it work with 1.7? (and yes, I know this is not exactly the topic of this discussion, but knowing the answer would help to know what to do with it)

**Comment by ttoine** [comment]
I looked at the file, it looks good to me.
what else can I do here ?

File: traces (hunk 1 above)
Comment (PierreRambaud) [reply traces:231]: Don't know, Don't know, should be seen with the @PrestaShop/product-team :)

File: traces [hunk 2]
```diff
@@ -37,6 +37,204 @@ const GITHUB_CONTRIBUTORS_POINT = 'https://api.github.com/repos/{repository}/con
...
+        'PrestaShop/community-theme-16',
+        'PrestaShop/performance-project',
+        'PrestaShop/engineering',
+        'PrestaShop/.github',
+    ]
```
Comment (ttoine) [inline traces:236]:
```suggestion
    'others' => [
        'PrestaShop/prestashop.github.io',
        'PrestaShop/community-theme-16',
        'PrestaShop/performance-project',
        'PrestaShop/engineering',
        'PrestaShop/.github',
    ]
```

now that solrsearch has been archived, let's remove it from the list.

**Comment by ttoine** [comment]
so is it possible to approve this pull request, now?

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @PierreRambaud & @ttoine

**Pull request merged**